packaging/arch/PKGBUILD.git

```diff
@@ -40,35 +44,39 @@ options=('!emptydirs' 'zipman' '!makeflags' 'docs')
 source=("${pkgname}::git+https://github.com/OpenSIPS/$_pkgname.git#branch=$_branch"
 	Makefile.conf.template)
 sha256sums=('SKIP'
-            '2eefae7d8d40259d9a0a4a6554574cf94418ea033d3a69936e928a52bcfcca63')
+	    '2d6114c21a9bd59bfaac755ec21568fefa4f64f356dc2d1bc8610476f7cc6ca8')
+validpgpkeys=(
+		'1EC4BE4FF2A6C9F4DDDF30F33C5F485DBD250D66' # Ralf Zerres (Package Signing)
+	)
 
 prepare() {
 	cd "$srcdir"/${pkgbase}
 
-	if [ -h "$srcdir"/Makefile.conf.devel.template ]; then
+	# check out given branch and update to head
+	git checkout $_branch
+	echo "prepare: pull $_branch"
+	git pull --rebase
+
+	if [ -h "$srcdir"/Makefile.conf.template ]; then
 		msg2 "preset Makefile.conf template"
-		test ! -f Makefile.conf.devel.template.orig  &&
+		test ! -f Makefile.conf.template.orig  &&
 		mv Makefile.conf.template Makefile.conf.template.orig
-		cp "$srcdir"/Makefile.conf.devel.template Makefile.conf.template
+		cp "$srcdir"/Makefile.conf.template Makefile.conf.template
 	fi
 
 	# patching
 	if [ ! -f .makepkg-patched ]; then
-		echo "  -> patching: "
-		git am --signoff ../../patches-git/0000-packaging-support-arch-linux.patch
-		git am --signoff ../../patches-git/0001-osp-Fix-module-compilation.patch
-		git am --signoff ../../patches-git/0002-opensips_json_c_helper.h-fix-preprocessor-definition.patch
-		git am --signoff ../../patches-git/0003-Makefile.defs-adapt-default-settings-for-XSL-parser.patch
-		git am --signoff ../../patches-git/0004-doxygen-update-element-list.patch
-		echo "  -> patching [done]"
+		msg2 "patching:"
+		git am --signoff ../../patches-git/0001-packaging-Arch-Linux-update-search-path-for-docbook.patch
+		#git am --signoff ../../patches-git/0002-packaging-Arch-Linux-update-PKGBUILD.git.patch
```

Comment:
These will probably not be needed, since we will be pushing the patches upstream.